pkg/controller/certificaterequests/venafi/venafi_test.go

```diff
@@ -27,6 +27,10 @@ import (
 	"testing"
 	"time"
 
+	"github.com/jetstack/cert-manager/pkg/issuer/venafi/client/api"
+
+	"github.com/jetstack/cert-manager/pkg/issuer/venafi/client"
```

Comment:
nit: Is the house style to put all the cert-manager imports last?

Reply:
I blame my IDE :P